### PR TITLE
BACKLOG-11734: Add category selectorTypes

### PR DESCRIPTION
**Pull request by FBerthelot**
## JIRA

https://jira.jahia.org/browse/BACKLOG-11734

## Tests

The following are included in this PR
- [ ] Unit Tests (Most changes _should_ have unit tests)
- [ ] Integration Tests

## Checklist

I have considered the following implications of my change: 

- [ ] Security (in particular for changes to authentication, authorization, data fetching, ...)
- [ ] Performance
- [ ] Migration
- [ ] Code maintainability

## Documentation

- [ ] Inline documentation
- [ ] Internal Documentation (wiki)
- [ ] User-facing Documentation

### CODE REVIEW

File: src/javascript/EditPanel/EditPanelContent/FormBuilder/Section/FieldSet/Field/SelectorTypes/Category/Category.jsx [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+                    {label: 'Ladoke Akintola University of Technology'},
+                    {label: 'Kohat University of Science and Technology  (KUST)'},
+                    {label: 'Hvanneyri Agricultural University'}
+                ]
+            }
+        ]
+    }
+];
+
+// eslint-disable-next-line
```
Comment (Taybou) [inline src/javascript/EditPanel/EditPanelContent/FormBuilder/Section/FieldSet/Field/SelectorTypes/Category/Category.jsx]: what's this?

File: src/javascript/EditPanel/EditPanelContent/FormBuilder/Section/FieldSet/Field/SelectorTypes/Category/Category.jsx [hunk 2]
```diff
@@ -0,0 +1,97 @@
+import React from 'react';
+import * as PropTypes from 'prop-types';
+import {FieldPropTypes} from '~/EditPanel/FormDefinitions/FormData.proptypes';
+import {FastField} from 'formik';
+import {DropdownTreeSelect} from '~/DesignSystem/DropdownTreeSelect';
+
+const data = [
```
Comment (Taybou) [inline src/javascript/EditPanel/EditPanelContent/FormBuilder/Section/FieldSet/Field/SelectorTypes/Category/Category.jsx]: this data should stay here?

File: src/javascript/EditPanel/EditPanelContent/FormBuilder/Section/FieldSet/Field/SelectorTypes/Category/Category.jsx (hunk 1 above)
Comment (FBerthelot) [reply src/javascript/EditPanel/EditPanelContent/FormBuilder/Section/FieldSet/Field/SelectorTypes/Category/Category.jsx]: I'm not using each props, but I will in the next PR. Don't worry :)

File: src/javascript/EditPanel/EditPanelContent/FormBuilder/Section/FieldSet/Field/SelectorTypes/Category/Category.jsx (hunk 2 above)
Comment (FBerthelot) [reply src/javascript/EditPanel/EditPanelContent/FormBuilder/Section/FieldSet/Field/SelectorTypes/Category/Category.jsx]: Not at all BACKLOG-11737 will remove this code :)

**Pull request merged**